docs/roadmap/cosmos-hub-roadmap-2.0.md

```diff
@@ -36,6 +36,8 @@ The upgrades aim to add features such as liquidity, economic security, usability
 - Gaia v7.0.x
 - Cosmos SDK v0.45
   - Minimal update with small fixes
+- Liquidity v1.4.4
```

Comment:
```suggestion
- Gravity DEX: Liquidity v1.4.4
```

Reply:
Hi, Thanks for the update, cosmos-sdk v0.45.0 based Liquidty is in progress at https://github.com/Gravity-Devs/liquidity/pull/10, which is expected to be v1.4.5.

Reply:
thanks for clarification @dongsam!